code/game/gamemodes/cult/blood_magic.dm

```diff
@@ -560,7 +566,7 @@
 
 /obj/item/melee/blood_magic/shackles/proc/CuffAttack(mob/living/carbon/C, mob/living/user)
 	if(!C.handcuffed)
-		playsound(loc, 'sound/weapons/cablecuff.ogg', 30, TRUE, -2)
+		playsound(loc, 'sound/weapons/cablecuff.ogg', 30, TRUE, -10)
```

Comment:
Shouldnt this be `SOUND_RANGE_SET(7)`? Its been a while since I've looked at this so im not sure